Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -348,7 +348,7 @@
           "SingleLineRegexes": [
             "$SEC101/037.SqlCredentialsAdoId",
             "$SEC101/037.SqlCredentialsAdoHost",
-            "$SEC101/037.SqlCredentialsAdoPort",
+            "?SEC101/037.SqlCredentialsAdoPort",
```

Comment:
>SEC101 [](http://example.com/codeflow?start=14&length=6)

@hulonjenkins, please observe, you can now specify an optionally populated value in a single line regex. the absence of this capability is a bug you identified several weeks ago.